packages/ag-charts-community/src/chart/dom/domStyles.css

```diff
@@ -72,7 +125,7 @@
 
     /* Focus Indicator */
     --ag-charts-focus-border-color: var(--ag-input-focus-border-color, var(--ag-charts-active-color));
-    --ag-charts-focus-border: solid 2px var(--ag-charts-focus-border-color);
+    --ag-charts-focus-border: solid 1px var(--ag-charts-focus-border-color);
```

Comment:
I changed this because this is what we'll use in all the form elements etc., and just override the width in the focus indicator for chart elements